addons/website_sale_collect_wishlist/static/src/js/wishlist.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+
+import publicWidget from '@web/legacy/js/public/public_widget';
+import '@website_sale_wishlist/js/website_sale_wishlist';
+import { rpc } from "@web/core/network/rpc";
+
+
+publicWidget.registry.ProductWishlist.include({
+    
+    events: Object.assign({}, publicWidget.registry.ProductWishlist.prototype.events, {
+        'click .o_update_wishlist': '_onClickUpdateWishlist',
+    }),
+
+    // #=== EVENT HANDLERS ===#
+    _onClickUpdateWishlist: async function (ev) {
+        this._addNewProducts($(ev.currentTarget));
+        this._onClickRemoveWishlist(ev);
+        window.location.reload();
+    },
+
+    /**
+     * Remove a product from the cart.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    async _onClickRemoveWishlist(ev) {
+        await rpc('/shop/cart/update_json', {
+            product_id: parseInt(ev.target.dataset.productId, 10),
```

Comment:
why 10? :eyes: 
(why we  need to round product id)

Reply:
The `10 `in `parseInt(ev.target.dataset.productId, 10)` specifies the decimal radix, ensuring consistent parsing of the product ID. I included it to keep things consistent with the `website_sale_collect` module, but since parseInt uses 10 by default, I guess we could skip it.

Reply:
But productId will always be in the decimal or there are some cases where productId is 5.5 :smile:

Reply:
I included the `10`  purely for consistency with the other modules like `website_sale_collect` and `website_sale_comparison_wishlist`, but since productId will always be a whole number, we can definitely skip that extra code. And maybe we can clean the code from other modules later on! :wink: